app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: callable) -> callable:
+    dict_cache = {}
+
+    def inner(*args, **kwargs) -> int:
+        key = (args, tuple(kwargs.items()))
```

Comment:
Check your key in debugger
![image](https://github.com/mate-academy/py-cache-decorator/assets/73850362/9bc1b4ed-835e-4574-8489-56973ffbab13)